src/pages/blog/airship-update-december-2020.md

```diff
@@ -0,0 +1,134 @@
+  https://www.youtube.com/watch?v=S0tJapmYNP4) (Digambar Patil, John Williams, Sirisha Gopigiri)
+* [China Mobile software and hardware integrated portable cloud platform test system](
+  https://www.youtube.com/watch?v=8N5PeW2l7Lc) (Xiaoguang Zhang, Zhiqiang Yu)
+* [hardware automation deployment and test with big scale in ChinaMobile NFV cloud](
+  https://www.youtube.com/watch?v=omKxP-qiKng) (Xiaoguang Zhang, Zhiqiang Yu)
+
+<br>
+
+## **AIRSHIP CONTAINER AS A SERVICE**
+
+Two operators will manage the lifecycle of Virtual Machines and their relationship to the cluster: vNode-Operator (ViNO)
+and the Support Infra Provider (SIP). These operators are new projects to the Airship ecosystem and were migrated to
+OpenDev earlier this month.
+
+So what do each of these operators do, at a thousand-foot view?
+
+[ViNO](https://opendev.org/airship/vino) (Virtual Node Operator)
+- Lays down libvirt and Virtual Machine definitions across bare-metal hosts
+- Fronts the Virtual Machines with sushy-tools for Redfish interactions via Metal3 et al.
+- Generates BareMetalHost definitions for the Virtual Machines
+
+<br>
+
+[SIP](https://opendev.org/airship/sip) (Support Infrastructure Provider)
+- Creates a load balancer
+- Provides utilities such as a jump pod
+- Facilitates CAPI (Cluster API) scheduling, e.g. "I need VMs with certain characteristics"
+
+<br>
+
+These projects are related to each other through CAPI, which will provision via Metal3 against the VMs created by ViNO
+and labeled by SIP.
+
+Look forward to more information on both of these projects in an upcoming blog post by Pete Birley, expected in January.
+In the meantime, learn more about these projects which were both introduced at the PTG in October through the recording
+below:
+
+- [Recording](https://zoom.us/rec/share/l3znN5H6JCvTnem-yaF0oUBJswd15a1k3vnQ-zvz93QaKclo2tdGoOU-5X7kluFE.idTvzC-AKnRpSm78?startTime=1603976835000) (Starts at about 3hr27m into the recording)
+- Password: ptg2020!
+
+<br>
+
+## **AIRSHIP USER SURVEY**
+
+Are you evaluating Airship or using Airship in production? We want to learn from your experience! Take the [_Airship
+User Survey_](https://www.surveymonkey.com/r/YKZ9NC2); the Working Committee reviews each response and helps us grow
+and mature Airship. Take the [_Airship User Survey_](https://www.surveymonkey.com/r/YKZ9NC2) today to be included in
+the next round of analysis.
+
+<br>
+
+## **AIRSHIP 2.0 PROGRESS**
+
+The progress shown below is for [_airshipctl_](https://opendev.org/airship/airshipctl), the new Airship 2.0 client.
+
+Last month, [_airshipctl_](https://opendev.org/airship/airshipctl) saw the following activity:
+
+* 15 authors
+* 31 commits
+* 281 files changed
+* 11,430 additions
+* 817 deletions
+* 11 closed issues
+* 19 new issues
+
+<br>
+
+This activity is contributing to the General Availability (GA) milestone. Below is the overall status of the GA
+milestone:
+
+![](/images/ga_status_december_2020.png)
+
+<br>
+
+## **GET INVOLVED**
```

Comment:
Let's include a small snippet introducing our new labels in our git issues.

Reply:
done.